package.json

```diff
@@ -42,7 +46,23 @@
     "sinon": "^2.1.0"
   },
   "scripts": {
-    "test": "mocha --require coffee-script/register --compilers coffee:coffee-script --compilers js:babel-core/register test/*.js",
-    "watch:test": "npm t -- --watch"
+    "test": "xo --quiet && mocha --require coffee-script/register --compilers coffee:coffee-script --compilers js:babel-core/register test/*.js",
```

Comment:
Why `--quiet`? let it be noisy :)

Reply:
@zelima there is no option like `noisy`. `quiet` uses for showing only errors and no warnings. I can remove `--quiet` option if you want

Reply:
@Mikanebu I know, I meant removing it :), but ok, let's keep it